### PR TITLE
fix: reset menu item

### PR DESCRIPTION
**Pull request by vik-13**

### CODE REVIEW

File: src/electron/menu/dev_menu_template.ts [hunk 1]
```diff
@@ -4,10 +4,6 @@ export const devMenuTemplate = (): MenuItem | MenuItemConstructorOptions => {
   return {
     label: 'Development',
     submenu: [
```
Comment (mgechev) [inline src/electron/menu/dev_menu_template.ts:6]: Why did you remove it from the menu?

Comment (vik-13) [reply src/electron/menu/dev_menu_template.ts:6]: It does the same as the reset from the main menu, just without confirmation dialog. Do you think it can be useful?

Comment (mgechev) [reply src/electron/menu/dev_menu_template.ts:6]: That's ok.
